kpi/mixins/object_permission.py

```diff
@@ -783,11 +780,13 @@ def __get_all_object_permissions(object_id):
 
     @staticmethod
     @cache_for_request
-    def __get_all_user_permissions(user_id):
+    def __get_all_user_permissions(user_id: int, asset_ids: list = []) -> dict:
```

Comment:
Let's use an immutable default just in case some future developer modifies `asset_ids` inside the function